region/crateria/east/Bowling Alley Path.json

```diff
@@ -228,12 +228,81 @@
     {
       "id": 9,
       "link": [1, 1],
-      "name": "Waver and Choot Farm",
+      "name": "Waver and Choot Farm (Both Doors)",
       "requires": [
-        {"resetRoom": {
-          "nodes": [1, 2]
-        }},
-        {"refill": ["Energy", "Missile", "Super"]}
+        {"resetRoom": {"nodes": [1]}},
+        {"resetRoom": {"nodes": [2]}},
+        {"or": [
+          {"and": [
+            "ScrewAttack",
+            {"cycleFrames": 990}
+          ]},
+          {"and": [
+            "Plasma",
+            {"cycleFrames": 620}
+          ]},
+          {"and": [
+            "Wave",
+            {"cycleFrames": 920}
+          ]},
+          {"and": [
+            "Spazer",
+            {"cycleFrames": 1010}
+          ]},
+          {"and": [
+            {"or": [
+              "canDodgeWhileShooting",
+              "Ice"
+            ]},
+            {"cycleFrames": 1740}
+          ]}
+        ]}
+      ],
+      "farmCycleDrops": [
+        {"enemy": "Waver", "count": 6},
+        {"enemy": "Choot", "count": 6}
+      ]
+    },
+    {
+      "link": [1, 1],
+      "name": "Waver and Choot Farm (Single Door)",
+      "requires": [
+        {"resetRoom": {"nodes": [1, 2]}},
+        {"cycleFrames": 100},
+        {"or": [
+          {"and": [
+            "ScrewAttack",
+            {"cycleFrames": 495}
+          ]},
+          {"and": [
+            "Plasma",
+            {"cycleFrames": 310}
+          ]},
+          {"and": [
+            "Wave",
+            {"cycleFrames": 460}
+          ]},
+          {"and": [
+            "Spazer",
+            {"cycleFrames": 505}
+          ]},
+          {"and": [
+            {"or": [
+              "canDodgeWhileShooting",
+              "Ice"
+            ]},
+            {"cycleFrames": 870}
+          ]}
```

Comment:
```suggestion
          ]},
          {"and": [
            "canUseGrapple",
            {"cycleFrames": 370}
          ]},
```

Reply:
Good catch, I've been forgetting about Grapple in general when working out these strats, need to remember it going forward.